src/hotspot/share/c1/c1_LIRGenerator.cpp

```diff
@@ -1251,8 +1244,13 @@ void LIRGenerator::do_isInstance(Intrinsic* x) {
     __ null_check(clazz.result(), info);
   }
 
+#ifdef HAVE_PD_C1_IS_INSTANCE_OF_STUB
```

Comment:
Is it the first occurrence of optional stub on C1? Alternatively, you could provide a default implementation on all other platforms which simply calls into `Runtime1::is_instance_of`.

Also, the macro doesn't look pretty here. `c1_Defs_*.hpp` promote a different pattern (a constant). I understand that it requires changing all `c1_Defs_*.hpp` files, but it would definitely look cleaner here.